src/components/Carousel/index.tsx

```diff
@@ -0,0 +1,252 @@
+  const carouselWidthSize = size
+  const translateValueOfLastImage = carouselWidthSize * (images.length - 1)
+  const arrowRight = ICON.CHEVRON_RIGHT_40
+  const arrowLeft = ICON.CHEVRON_LEFT_40
+  const [translateValue, setTranslateValue] = useState<number>(0)
+  const [startClientX, setStartClientX] = useState<number>(0)
+  const [endClientX, setEndClientX] = useState<number>(0)
+  const [cursorOn, setCursorOn] = useState<boolean>(false)
+
+  const handleIndicator = (imageIndex: number): void => {
+    setTranslateValue((imageIndex - 1) * carouselWidthSize)
+  }
+
+  const TRANSLATE_VALUE_OF_NAV_TYPE = {
+    LEFT: -carouselWidthSize,
+    RIGHT: carouselWidthSize
+  }
+
+  const handleOffset: HandleOffset = navType => {
+    const { LEFT } = NAV_TYPE
+    const endPoint = navType === LEFT ? 0 : translateValueOfLastImage
+    const prevTranslateValue = navType === LEFT ? translateValueOfLastImage : 0
+    if (translateValue === endPoint) {
+      setTranslateValue(prevTranslateValue)
+    } else {
+      setTranslateValue(prev => prev + TRANSLATE_VALUE_OF_NAV_TYPE[navType])
+    }
+  }
+
+  const handleMouseDown = (e: { clientX: SetStateAction<number> }): void => {
+    setStartClientX(e.clientX)
+    setCursorOn(true)
+  }
+
+  const handleMouseUp = (e: { clientX: SetStateAction<number> }): void => {
+    setEndClientX(e.clientX)
+    setCursorOn(false)
+  }
+  const handleTouchStart: TouchEventHandler<HTMLDivElement> = e => {
+    setEndClientX(e.touches[0].clientX)
+    setCursorOn(true)
+  }
+  const handleTouchStartEnd: TouchEventHandler<HTMLDivElement> = e => {
+    setStartClientX(e.changedTouches[0].clientX)
+    setCursorOn(false)
+  }
+
+  useEffect(() => {
+    const dragSpace = Math.abs(startClientX - endClientX)
+    const userSlideRight = endClientX < startClientX && dragSpace > 100
+    const userSlideLeft = endClientX > startClientX && dragSpace > 100
+    if (startClientX === 0) {
+      return
+    }
+    if (userSlideRight) {
+      handleOffset('RIGHT')
+    } else if (userSlideLeft) {
+      handleOffset('LEFT')
+    }
+  }, [endClientX])
+
+  return (
+    <StyledCarouselWrapper>
+      <StyledSlider
+        cursorOn={cursorOn}
+        size={carouselWidthSize}
+        onMouseDown={handleMouseDown}
+        onMouseUp={handleMouseUp}
+        onTouchEnd={handleTouchStartEnd}
+        onTouchStart={handleTouchStart}>
+        <StyledImageBox translateValue={translateValue || null}>
+          {images.map(image => {
+            return (
+              <StyledImage
+                key={image.id}
+                alt={`${name}- ${image.id}`}
+                size={carouselWidthSize}
+                src={image.url}
+              />
+            )
+          })}
+        </StyledImageBox>
+        {isArrow && (
+          <StyledArrowBox>
+            <StyledArrow
+              alt="arrow-left"
+              src={arrowLeft}
+              onClick={(): void => {
+                handleOffset('LEFT')
```

Comment:
```suggestion
                handleOffset(NAV_TYPE.LEFT)
```
요기도 상수 이용하는 게 좋을 것 같습니다!